rhscl-postgresql-apb/roles/rhscl-postgresql-apb-openshift/tasks/main.yml

```diff
@@ -0,0 +1,109 @@
+- name: create namespace
+  openshift_v1_project:
+    name: '{{ namespace }}'
+
+- name: create volumes
+  k8s_v1_persistent_volume_claim:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    state: present
+    access_modes:
+      - ReadWriteOnce
+    resources_requests:
+      storage: 1Gi
+
+- name: create service
+  k8s_v1_service:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    state: present
+    labels:
+      app: postgresql-demo-apb
+      service: postgresql
+    selector:
+      app: postgresql-demo-apb
+      service: postgresql
+    ports:
+    - name: port-5432
+      port: 5432
+      protocol: TCP
+      target_port: 5432
+  register: postgres_service
+
+- set_fact:
+    image: postgresql-{{ postgresql_version | replace('.', '') }}-rhel7
```

Comment:
Could you move this from a set_fact to a variable in the defaults? So far as I can tell it doesn't need to be in a task